proposals/2022-ambassadors.md

```diff
@@ -0,0 +1,55 @@
+# Proposal for budget and process for OWASP Ambassadors 
+
+## What is the problem we are trying to solve
+
+It seems that there is limited understanding of OWASP around the wider developer community. Whilst people may have heard of the top 10, it is not clear that they are familar with other projects which could help them or that they can/should be more involved in OWASP events.
+
+## Aim
+
+To increase awareness and understanding of OWASP among the wider developer community.
+
+In particular:
+* Encourage new members to join by highlighting the benefits
+* Encourage new corporate supporters by highlighting the importance of the organization
+* Encourage greater involvement in the local chapter
+* Showcase the variety of projects and resources which OWASP provides
+
+## Proposal Summary
+
+Creation of an OWASP ambassador role who would attend meetups and conferences and provide more in-depth information about getting into OWASP. 
+
+## Proposal Details
+
+To qualify as an OWASP ambassador, the person should commit to going to four non-OWASP events per year with the aim of advocating for OWASP and track this for audit on renewal. This would be a purely volunteer position and new applicants would have to be able to demonstrate previous OWASP involvement as well as having OWASP membership.
+
+In return, the ambassador would receive:
+* Specific recognition and thanks
+* Supplies of very basic/inexpensive OWASP swag such as stickers or pens to give out
+* Some dedicated OWASP apparel to make them stand out.
+* Coordination with the local chapter
+
+### OWASP stall
```

Comment:
Should this be staff?

Reply:
No but I will changed the wording to clarify